content/devices/angelica.md

```diff
@@ -76,7 +76,7 @@ Please download the belowed needed files and tools:
     - The first boot may take longer, and at least one spontaneous reboot is expected during the process
     - You should be greeted with the lock screen, the default password is `1234`
 
-Congratulations, if everything went well, now you should be running Droidian.
```

Comment:
The files inside content/devices are now generated by the generate_device_pages.js script, if the changes are present in template files, the changes shoul relect here.

Reply:
i changed this piece of text in templates/* then ran generate_device_pages.js